2018-projects.md

```diff
@@ -0,0 +1,197 @@
+
+#### Made by Joseph Gabay, Axel Simms, Emily Hontoria, and Anjaneya Naik ####
+
+Using a webcam, this app takes a picture of your face and uses the Indico API to detect your mood. From here, it matches your mood with a song from a pool of over 180 top songs that have had their lyrics analyzed with the Indico API. It then plays your 'mood song' through spotify.
+
+### [Lucid VCS](https://github.com/caseyz/lucid) ###
+
+#### Made by Milo Turner, Zach Casey, and Kira Evans ####
+
+Version control system, controlled by interacting with a text file on emacs, uses "patch theory" to calculate merge, commute etc.
+
+### [Dadabots](http://github.com/cortexelus/nofxbot) ###
+
+#### Made by CJ Carr, Zack Zukowski, and Bryan Robinson ####
+
+Used a recurrent neural network to train a raw-audio model on punk band NOFX, generated a song that definitely sounds like NOFX but with nonsense lyrics. 
+
+### [Get Together](https://www.github.com/donnellycolton/hbp2018front) ###
+
+#### Made by Colton Donelly, Aditya Jalan, and Daniel Melcer####
+
+An event management app which covers every aspect of an event, 'from organizing an event to reaching the venue on time' The app, after 2 nights of work, boasts of a login page with persistent login capability and secure password storage on server, as well as a task assignment functionality, aimed at making an event more efficient by distributing tasks among the attendees. While the client side of the application does not yet support task assignment, the server is prepared to assign tasks within the group to specific members or to the group as a whole. The application gives the user the control to create his own private 'party-space' to which he can invite other members through a personalized code, and assign tasks and requirements for the party to fellow attendees. 
+
+Future prospective improvements include but are not limited to event decisions within events, for e.g a Pepsi v/s Coke poll; Venue mapping and Uber/Lyft integration within the App; 'Wishlist' integration; option to look up and buy tickets to public events; dedicated group chats and inter-personal chat bubbles; calendar integration for appropriate reminders of events; and even the functionality of booking venues through the app, in the long run, among tons of other features. 
+
+### [CourseOn](https://github.com/PriscaJ/HackBeanpot2018.git)  ###
+
+#### Made by Prisca Joseph, Sarah Wessel, Rucha Khanolkar, and Megan Lau ####
+
+CourseOn is a web application to help students to organize which courses they need to take throughout their time at Northeastern. This allows students to generate their grad plan easily, dynamically, and interactively.
+
+### [pupDate](https://github.com/annachoi4/pupDate) ###
+
+#### Made by Anna Choi, Andres Rodriguez, Gabriella Roman, and Christopher Trinh ####
+
+A web application for dog owners to find their dogs some nearby playmates, since Anna owns a dog and is always looking for playmates! Made in PyCharm.
```

Comment:
somethings wrong here, desc displays as large header text

Reply:
@applebynj old cruft here. Look at L166. H1's in MarkDown are also triggered by a line after of 3-or-so hyphens.

```md
example
----
```
turns to 
example
---

Reply:
ahh thank you for catching that - i had put those in there to help differentiate between copy and sections i'll remove that!